packages/manager/modules/pci/src/projects/project/instances/add/add.controller.js

```diff
@@ -68,6 +69,12 @@ export default class PciInstancesAddController {
     };
     this.selectedPrivateNetwork = this.defaultPrivateNetwork;
     this.availablePrivateNetworks = [this.defaultPrivateNetwork];
+    this.automatedBackup = {
+      available: this.ovhSubsidiary !== 'US',
```

Comment:
It's ok here because currently only US sub is included in US region, but what if it's another sub using the US manager ?

Reply:
should we use Environment.region from manager-core then?

Reply:
I was thinking`coreConfig.getRegion()` instead